grpc-gateway/service/updateResource_test.go

```diff
@@ -286,7 +286,6 @@ func TestRequestHandlerRunMultipleUpdateResource(t *testing.T) {
 	defer cancel()
 
 	raCfg := raTest.MakeConfig(t)
-	raCfg.Clients.Eventstore.SnapshotThreshold = 5
```

Comment:
What changed with the SnapShot behavior? (I suspect it will be simpler to explain this on SM tomorrow, so just take this as a reminder).